consensus/src/pubkeys.rs

```diff
@@ -64,6 +65,9 @@ pub enum InvalidPubkey<const LEN: usize> {
 )]
 pub struct CompressedPk(PublicKey);
 
+impl StrictSerialize for CompressedPk {}
+impl StrictDeserialize for CompressedPk {}
```

Comment:
This is not needed.

Strict(De)Serialize used to serialize the structure as a whole to a file. Why would you need it?

Reply:
We implemented a schema that uses CompressedPk as a global value, this is why we need it.